### PR TITLE
project: private marketplace docs

### PR DESCRIPTION
**Pull request by GiovannaMonti**
## Pull Request Type

- [x] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [x] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [x] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [x] No sensible content has been committed

### CODE REVIEW

File: docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ The following table describes the capabilities that are granted when assigning a
 | View Company Clusters information                                      | `console.company.cluster.view`                         | ✅     | ✅       | ✅        | ✅         | ✅                    | ✅            |
 | Manage Company Project Templates                                       | `console.company.templates.manage`                     |        |          |           |            |                       | ✅            |
 | Manage Company Views in Design Overview                                | `console.company.configuration.views.manage`                          |        |           |          |              |  ✅  |  ✅  |
+| Manage all Marketplace resources                                     | `console.root.marketplace.manage`                          |        |           |          |              |   |   |
```
Comment (fredmaggiowski) [inline docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md:68]: this should go with root-level permissions

File: docs/marketplace/add_to_marketplace/contributing_overview.md [hunk 2]
```diff
@@ -46,6 +59,7 @@ Here below are listed all the properties that must be provided for each type of
 - **categoryId**: a label to help categorize components by their purpose or use case. As specified before, categories are only created internally at Mia-Platform. The `categoryId` of a component uniquely determines both a specific category and a sub-category (e.g. Start from Code (category) - Node.js (subcategory) will be identified by the `categoryId` "nodejs")
 - **supportedBy**: a label to identify the company that has produced the component (only used if `supportedByImage` is not provided)
 - **image** and **supportedByImage**: respectively the image that will be associated with the component and the image that will be associated with the company that has produced it.
+- **tenantId** (optional): the Company id, only needed if the component is a private Marketplace component that must not be generally available on the Mia-Platform Marketplace.
```
Comment (fredmaggiowski) [inline docs/marketplace/add_to_marketplace/contributing_overview.md:62]:
```suggestion
- **tenantId** (optional): the Company id, only needed if the component is a private, Company-owned, Marketplace component that must not be generally available on the Mia-Platform Marketplace.
```

File: docs/marketplace/add_to_marketplace/contributing_overview.md [hunk 3]
```diff
@@ -4,13 +4,26 @@ title:  Add a new element to the Marketplace
 sidebar_label: Add a new element to the Marketplace
 ---
 
-:::info
-If you want to contribute to Mia-Platform Marketplace with a new component, you can [open an issue](https://github.com/mia-platform/community/issues/new?labels=marketplace&template=marketplace-contribution.md&title=Add+new+marketplace+item), wherein you will find what information is needed for the request.
+In this section, we dive into the process of creating a new component for the Marketplace, analyzing the main differences between different types of components and providing some configuration examples.
+
```
Comment (fredmaggiowski) [inline docs/marketplace/add_to_marketplace/contributing_overview.md:8]: Let's add a note that marketplace items can only be created by people with access to the Mia-Platform Console CMS, therefore you may have to ask your Mia-Platform referent to add your marketplace items for you.

**Review by fredmaggiowski — CHANGES_REQUESTED** [review]

File: docs/marketplace/add_to_marketplace/contributing_overview.md [hunk 4]
```diff
@@ -4,13 +4,26 @@ title:  Add a new element to the Marketplace
...
+However, a Company might want to have its own private Marketplace, with private components that must not be shared with everyone else on the Console. 
+
+To achieve this, a Company Owner or a Project Administrator can create new private components **from the CMS** by specifying their Company's identifier in the `tenantId` field.
+
+Company Owners and Project Administrators can create, delete and update Company-wide markeplace elements, but they cannot manage public Marketplace items.  
+The Console Super User role, instead, has permission to manage all Marketplace items from the CMS without any limitations.
+
+If you want to contribute to the **public Mia-Platform Marketplace** with a new component instead, you can [open an issue](https://github.com/mia-platform/community/issues/new?labels=marketplace&template=marketplace-contribution.md&title=Add+new+marketplace+item) on GitHub, wherein you will find what information is needed for the request.
```
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/contributing_overview.md:20]:
```suggestion
If you want to contribute to the **public Mia-Platform Marketplace** with a new component instead, and make it available for everyone, you can [open an issue](https://github.com/mia-platform/community/issues/new?labels=marketplace&template=marketplace-contribution.md&title=Add+new+marketplace+item) on GitHub, wherein you will find what information is needed for the request.
```

File: docs/marketplace/add_to_marketplace/contributing_overview.md (hunk 2 above)
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/contributing_overview.md:62]:
```suggestion
- **tenantId** (optional): the Company id, only needed if the component is a private Marketplace component for your Company and therefore must not be generally available on the public Mia-Platform Marketplace.
```

File: docs/marketplace/add_to_marketplace/contributing_overview.md [hunk 5]
```diff
@@ -46,6 +63,7 @@ Here below are listed all the properties that must be provided for each type of
 - **categoryId**: a label to help categorize components by their purpose or use case. As specified before, categories are only created internally at Mia-Platform. The `categoryId` of a component uniquely determines both a specific category and a sub-category (e.g. Start from Code (category) - Node.js (subcategory) will be identified by the `categoryId` "nodejs")
 - **supportedBy**: a label to identify the company that has produced the component (only used if `supportedByImage` is not provided)
 - **image** and **supportedByImage**: respectively the image that will be associated with the component and the image that will be associated with the company that has produced it.
+- **tenantId** (optional): the Company id, only needed if the component is a private, Company-owned, Marketplace component that must not be generally available on the Mia-Platform Marketplace.
```
Comment (GiovannaMonti) [inline docs/marketplace/add_to_marketplace/contributing_overview.md:66]:
```suggestion
- **tenantId** (optional): the Company id, only needed if the component is a private, Company-owned, Marketplace component that must not be generally available on the public Mia-Platform Marketplace.
```

**Review by fredmaggiowski — APPROVED** [review]

**Pull request merged**